.github/workflows/publish-pypi-on-tag.yml

```
@@ -68,6 +68,7 @@ jobs:
     if: >-
       github.ref_type == 'tag' &&
       !startsWith(github.ref_name, 'actions-v')
+    environment: pypi
```

actionlint (1.7.12)

[error] when a reusable workflow is called with "uses", "environment" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "pypi"

(syntax-check)